README.md

```diff
@@ -462,6 +462,22 @@ keys are delegated attributes' names and values are keys on synced data Hash. Th
 version of `delegate :name, to: :synced_data` which works with Hash reserved attributes names, like
 `:zip`, `:map`.
 
+## Persisting fetched objects
```

Comment:
well, we allow people to choose how they want to fetch data, so I think it's worth mentioning, that choosing this will affect other settings (like yeah you can choose auto_paginate but it won't work anyway)
Or just force pagination with block and drop support for auto paginate